.github/workflows/format-test.yml

```
@@ -42,6 +42,12 @@ jobs:
       - uses: actions/checkout@v4
         with:
           lfs: true
+      - uses: actions/cache@v4
+        with:
+          path: ${{ matrix.projectPath }}/Library
```

actionlint (1.7.9)

property "projectpath" is not defined in object type {testmode: string; unityversion: string}

(expression)